routes.js

```diff
@@ -50,10 +50,11 @@ export default function getRoutes(
 	{
 		API_SERVER_ROOT_URL,
 		PHOTO_SCALER_SALT,
-	}) {
+	},
+	apiProxyFn$ = apiProxy$) {
```

Comment:
That said, I'm not super happy with the overall signature for this function now - I think we should probably refactor it to be a single config object arg at some point, but since that's a breaking change I don't want to do it now.